code/datums/antagonists/datum_wizard.dm

```diff
@@ -63,11 +63,11 @@
 		add_objective(escape_objective)
 
 /datum/antagonist/wizard/apply_innate_effects()
-	name_wizard()
 	forge_wizard_objectives()
 	var/mob/living/carbon/human/H = owner.current
-	H.equipOutfit(/datum/outfit/wizard)
+	H.equipOutfit(pick(/datum/outfit/wizard,/datum/outfit/wizard/red,/datum/outfit/wizard/weeb), summoner)
```

Comment:
`/mob/living/carbon/human/proc/equipOutfit(outfit, visualsOnly = FALSE)`

I think you wanted `H.equipOutfit(summoner ? pick(/datum/outfit/wizard,/datum/outfit/wizard/red,/datum/outfit/wizard/weeb) : /datum/outfit/wizard, FALSE)`